StringFormatterBench/TraceEvent.ReadMe.txt

```diff
@@ -0,0 +1,79 @@
+﻿
+************* Welcome to the  Microsoft.Diagnostics.Tracing.TraceEvent library!  ***************
```

Comment:
Do you think it's a good idea to commit these TraceEvent docs? If so, I'd prefer them moved somewhere like external/TraceEvent/